src/Sulu/Bundle/SecurityBundle/Admin/SuluSecurityContactsContentNavigationProvider.php

```diff
@@ -2,30 +2,36 @@
 
 namespace Sulu\Bundle\SecurityBundle\Admin;
 
-use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationInterface;
+use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationProviderInterface;
 use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationItem;
 use Sulu\Component\Security\Authorization\SecurityCheckerInterface;
 
-class SuluSecurityContactsContentNavigation implements ContentNavigationInterface
+class SuluSecurityContactsContentNavigationProvider implements ContentNavigationProviderInterface
 {
-    private $navigation = array();
+    /**
+     * @var SecurityCheckerInterface
+     */
+    private $securityChecker;
 
     public function __construct(SecurityCheckerInterface $securityChecker)
     {
-        if ($securityChecker->hasPermission('sulu.security.users', 'view')) {
+        $this->securityChecker = $securityChecker;
+    }
+
+    public function getNavigationItems(array $options = array())
+    {
+        $navigation = array();
+
+        if ($this->securityChecker->hasPermission('sulu.security.users', 'view')) {
```

Comment:
Don't know if return early is a good idea here. Makes things more complicated if there will be more items added without a security check later.